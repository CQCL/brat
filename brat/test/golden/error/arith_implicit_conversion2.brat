f(Int, Float) -> Int
f(x, y) = x + (y - x)
