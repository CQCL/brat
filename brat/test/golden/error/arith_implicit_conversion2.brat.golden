Error in test/golden/error/arith_implicit_conversion2.brat@FC {start = Pos {line = 2, col = 16}, end = Pos {line = 2, col = 17}}:
f(x, y) = x + (y - x)
               ^

  Type mismatch when checking y
Expected: (left :: Int)
But got:  (y :: Float)


